### ISSUE TITLE
Accessibility: action labels and custom actions not supported

### ISSUE THREAD

**Issue by ialokim** [JetBrains/compose-multiplatform#2118]
Slightly adapting the [example from the docs](https://github.com/JetBrains/compose-jb/tree/master/tutorials/Accessibility), by adding a `onClickLabel` and a [custom action](https://developer.android.com/jetpack/compose/accessibility#custom-actions).

```kotlin
fun main() = singleWindowApplication {
    var count by remember { mutableStateOf(0) }

    Box(modifier = Modifier.padding(50.dp)) {
        Box(modifier = Modifier
            .background(Color.LightGray)
            .fillMaxSize()
            .clickable(onClickLabel = "increment value") { count += 1 }
            .semantics {
                role = Role.Button
                contentDescription = "Click to increment value"
                customActions = listOf(
                    CustomAccessibilityAction("reset value") { count = 0; true }
                )
            }
        ) {
            val text = when (count) {
                0 -> "Click Me!"
                1 -> "Clicked"
                else -> "Clicked $count times"
            }
            Text(text, modifier = Modifier.align(Alignment.Center), fontSize = 24.sp)
        }
    }
}
```

Results in: click action still shown as "press" and custom action not accessible at all

<img width="563" alt="Bildschirmfoto 2022-06-15 um 12 46 22" src="https://user-images.githubusercontent.com/18088991/173809482-87e55baa-3912-44fa-a045-47ed7ec7351f.png">

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2118">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
